CosmicTrigger/include/Event.h

```diff
@@ -11,10 +11,13 @@ struct hit
   unsigned int time;
 };
 
-struct event
+class event
 {
-  unsigned int eventNumber;
-  time_t time;
-  vector<hit> measurements;
+public:
+    event():time(0), eventNumber(0), errorCode(-1){}
```

Comment:
I get a warning here when I compile: time should be initialized after eventNumber (in the same order as the declarations).